Codeinwp/optimole-wp
# feat: disallow sign up with duplicate domain [ref: #517]

### All Submissions:

 

### Changes proposed in this Pull Request:
Needs & Details at: https://github.com/Codeinwp/optimole-service/pull/1050
 
Closes https://github.com/Codeinwp/optimole-service/issues/517.

### How to test the changes in this Pull Request:

1. Once both Dashboard & WP are available for testing, create a new account from a website with some email. It should work fine.
2. Once that is done, try to disconnect and sign up from the same site using a new email, it should notify you that the website has already been used with Optimole.

### Other information:
